Modules/hpe3par_snapshot.py

```diff
@@ -545,6 +545,7 @@ def create_schedule(
                 cmd.append("-exp")
                 cmd.append(str(expiration_hours)+"h")
             if retention_time:
+                cmd.append("-f")
```

Comment:
When user inputs retention_time then ,it is showing below warning during snapshot creation and snapshot creation is getting failed. 
CSSOS-SSA05 cli% showtask -d 9953
  Id Type           Name          Status Phase Step -------StartTime------- ------FinishTime------- -Priority- -User--
9953 scheduled_task ar_ansible_sc failed   ---  --- 2018-11-26 17:00:00 IST 2018-11-26 17:00:00 IST n/a        3paradm

Detailed status:
2018-11-26 17:00:00 IST Created     task.
2018-11-26 17:00:00 IST Updated     Executing "ar_ansible_sc" as 1:36766
2018-11-26 17:00:00 IST Updated     Warning: The retention time cannot be removed or reduced once it is
2018-11-26 17:00:00 IST Updated     set. This volume cannot be removed until its retention time expires.
2018-11-26 17:00:00 IST Updated     Are you sure you want to set the retention time?
2018-11-26 17:00:00 IST Updated     select q=quit y=yes n=no:
2018-11-26 17:00:00 IST Error       Task exited with status 255
2018-11-26 17:00:00 IST Failed      Could not complete task.

hence added '-f' option in command for retention_time option.

Reply:
Results after fixing the issue:-
CSSOS-SSA05 cli% showsched
                                                                                                        ------ Schedule ------
SchedName     File/Command                                                                              Min Hour DOM Month DOW CreatedBy Status Alert NextRunTime
am_ansible_sc createsv -ro -exp 1h am_ansible_volume.@y@@m@@d@@H@@M@@S@ am_ansible_volume                 0    *   *     *   * 3paradm   active     Y 2018-11-26 20:00:00 IST
ar_ansible_sc createsv -ro -exp 2h -f -retain 1h al_ansible_volume.@y@@m@@d@@H@@M@@S@ al_ansible_volume   0    *   *     *   * 3paradm   active     Y 2018-11-26 20:00:00 IST
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
2             total
CSSOS-SSA05 cli%

CSSOS-SSA05 cli% showvv
                                                                                                                  -Rsvd(MiB)-- -(MiB)-
   Id Name                            Prov Compr Dedup Type  CopyOf                      BsId Rd -Detailed_State-   Snp    Usr   VSize
    1 .srdata                         full NA    NA    base  ---                            1 RW normal               0  81920   81920
    0 admin                           full NA    NA    base  ---                            0 RW normal               0  10240   10240
46394 al_ansible_volume               tpvv No    No    base  ---                        46394 RW normal             512    512    1024
46404  al_ansible_volume.181126180000 snp  NA    NA    vcopy al_ansible_volume          46394 RO normal              --     --    1024
46406  al_ansible_volume.181126190000 snp  NA    NA    vcopy al_ansible_volume          46394 RO normal              --     --    1024
46395 am_ansible_volume               tpvv No    No    base  ---                        46395 RW normal             512    512    1024
46403  am_ansible_volume.181126180000 snp  NA    NA    vcopy am_ansible_volume          46395 RO normal              --     --    1024
46407  am_ansible_volume.181126190000 snp  NA    NA    vcopy am_ansible_volume          46395 RO normal              --     --    1024

CSSOS-SSA05 cli% showtask
  Id Type             Name                                                   Status Phase Step -------StartTime------- ------FinishTime------- -Priority- -User--
9957 scheduled_task   ar_ansible_sc                                          done     ---  --- 2018-11-26 18:00:00 IST 2018-11-26 18:00:00 IST n/a        3paradm
9958 scheduled_task   am_ansible_sc                                          done     ---  --- 2018-11-26 18:00:00 IST 2018-11-26 18:00:00 IST n/a        3paradm
9961 scheduled_task   am_ansible_sc                                          done     ---  --- 2018-11-26 19:00:00 IST 2018-11-26 19:00:00 IST n/a        3paradm
9962 scheduled_task   ar_ansible_sc                                          done     ---  --- 2018-11-26 19:00:00 IST 2018-11-26 19:00:00 IST n/a        3paradm

CSSOS-SSA05 cli%
CSSOS-SSA05 cli% showvv -expired
                                                                                                        -Rsvd(MiB)- -(MiB)-
   Id Name                           Prov Compr Dedup Type  CopyOf             BsId Rd -Detailed_State-   Snp   Usr   VSize
46403 am_ansible_volume.181126180000 snp  NA    NA    vcopy am_ansible_volume 46395 RO normal              --    --    1024
46393 ar_ansible_volume.181126172036 snp  NA    NA    vcopy ar_ansible_volume 46388 RO normal              --    --    1024
---------------------------------------------------------------------------------------------------------------------------
    2 total                                                                                                 0     0    2048
CSSOS-SSA05 cli%
CSSOS-SSA05 cli%